forge-gui/res/cardsfolder/upcoming/luxurious_libation.txt

```diff
@@ -0,0 +1,9 @@
+Name:Luxurious Libation
+ManaCost:X G
+Types:Instant
+A:SP$ Pump | Cost$ X G | ValidTgts$ Creature | TgtPrompt$ Select target creature | NumAtt$ +X | NumDef$ +X | SubAbility$ DBToken | SpellDescription$ Target creature gets +X/+X until end of turn. Create a 1/1 green and white Citizen creature token.
```

Comment:
For targets that are simply a one word core type, you can omit TgtPrompt if you like... I made it auto-lower case them